### PR TITLE
Allow installing for TYPO3 v10 and fix AbstractViewHelper

### PR DESCRIPTION
**Pull request by joey-bolts**
Form Recaptcha seems to work out of the box.

### CODE REVIEW

**Comment by Cretection** [comment]
I test your Fork. At my installation it throws this:
`Fatal error: Declaration of Haffner\JhCaptcha\Validation\Validator\Powermail::spamCheck() must be compatible with In2code\Powermail\Domain\Validator\SpamShield\AbstractMethod::spamCheck(): bool in /usr/local/www/apache24/[***]/htdocs/typo3conf/ext/jh_captcha/Classes/Validation/Validator/Powermail.php on line 0`

**Comment by joey-bolts** [comment]
This was just an partial update, since i wasn't using all features the extension supports

File: ext_emconf.php [hunk 1]
```diff
@@ -11,10 +11,10 @@
...
-    'constraints' => array(
-        'depends' => array(
-            'typo3' => '8.7.0-9.5.99',
-        ),
-    ),
-);
+    'version' => '3.0.4',
+    'constraints' => [
+        'depends' => [
+            'typo3' => '8.7.0-10.5.99',
```
Comment (kitzberger) [inline ext_emconf.php]: There's no 10.5! 10.4.99 should do fine though.

**Review by kitzberger — APPROVED** [review]
Thanks for your efforts , it seems to be working just fine :-)

**Comment by TehTux** [comment]
Thank you for your PR. Please do not raise the version in PRs, so I can not merge it. You forgot a few things. I have made the necessary adjustments and will roll out a release soon.

**Pull request closed without merging**